### DISCUSSION TITLE
FAT32 vs exFAT   ive found real world uses for every non-firewire release pretty much, just curiouse about something?

### DISCUSSION THREAD

**Discussion by JMaxxL** [israpps/wLaunchELF_ISR#44] [General]
I really like the idea of not having HOST with no-network versions, especially when using my media remote because it is moves to fast.

I never use anything larger then 32GB on USB/mx4sio for homebrew with either  all my games run off SSD & are moved via OEM-SATA adapted modem. But is it possible to do a normal non-exFAT variant?

example:   "[BOOT-NO_NETWORK-MX4SIO.ELF]" on the next release ? 

instead of this one im using [BOOT-EXFAT-NO_NETWORK-MX4SIO.ELF] I may never us exFAT formated card on my phat console with my SSD installed

![adapter](https://github.com/israpps/wLaunchELF_ISR/assets/78325531/2c801f03-3c4c-40b8-a268-96517e4a369e)

**Comment by israpps** [comment]
hi

EXFAT builds should read FAT32 filesystems without issues.

So I dont see the need for separating mx4sio and EXFAT.

The reason why they're linked together is BDM modules

**Reply by JMaxxL** [reply]
this is best way to install games, over network right?
[HDL-Batch-installer](https://github.com/israpps/HDL-Batch-installer)

but is there a way to use USB or mx4sio to install games to hard drive from mx4sio or USB?


Is there a way to move games, I looked threw many apps I didnt see any ELF homebrew that really do that it all seems like its network based, or just run off USB or mx4sio
